packages/loader/src/Polyfill.ts

```
@@ -0,0 +1,33 @@
+          const matchFlag = flags.split("g").join("");
+          const matchPattern = new RegExp(pattern.source, matchFlag);
+
+          for (let index in matchResult) {
+            const item = matchResult[index];
+            yield item.match(matchPattern) as RegExpExecArray;
+          }
+        })();
+      };
+    }
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)